public/js/components/Header/Header.js

```diff
@@ -55,62 +62,48 @@ class Header extends React.Component {
   }
 
   renderPublishedState = () => {
-    if(this.props.atom) {
-
-      const atomPublishState = publishState(this.props.atom);
+    const atomPublishState = publishState(this.props.atom);
+    return (
+        <span className={`publish-state publish-state--${atomPublishState.id}`}>{atomPublishState.text}</span>
+    );
+  }
 
+  renderTakeDownButton = (atomPublishState) => {
+    if(atomPublishState.id !== 'draft') {
       return (
-        <nav className="main-nav" role="navigation">
-          <ul className="main-nav__list">
-            <li className="toolbar__item main-nav__item">
-              <span className={`publish-state publish-state--${atomPublishState.id}`}>{atomPublishState.text}</span>
-            </li>
-            <li className="toolbar__item main-nav__item">
-              {this.renderSaveState()}
-            </li>
-          </ul>
-        </nav>
+        <button type="button" disabled={atomPublishState.id === 'taken-down'} onClick={this.takeDownAtom} className="toolbar__item toolbar__button">Take down</button>
       );
     }
     return false;
   }
 
-  renderPublishButton = () => {
-    if(this.props.atom) {
+  renderHeaderRight = () => {
 
-      const atomPublishState = publishState(this.props.atom);
+    const atomPublishState = publishState(this.props.atom);
 
-      return (
+    return (
         <div className="toolbar__container">
-          <div className="toolbar__item">
-            {this.props.presence ? <PresenceIndicator presence={this.props.presence} /> : false}
-          </div>
-          <button disabled={atomPublishState.id === 'published'} type="button" onClick={this.publishAtom} className="toolbar__item toolbar__button">Publish</button>
-          {this.renderTakeDownButton(atomPublishState)}
+          {this.props.presence ? <PresenceIndicator presence={this.props.presence} /> : false}
```

Comment:
I know it was me, but we may have the same problem here as presence will still exist in state after visiting one page.